Modules/Project/Resources/views/subviews/edit-resource-requirement.blade.php

```diff
@@ -0,0 +1,35 @@
+<div class="card">
+  <div id="projectResourceRequirementForm">
+    <form  action="{{ route('project.update', $project) }}" method="POST" id="updateProjectResourceRequirement" enctype="multipart/form-data">
+      @csrf
+      <input type="hidden" value="project_resource_requirement" name="update_section">
+        <div class="card-body form-body">
+          <h3 class="mb-7 font-weight-bold">Requirement Details</h3>
+            <table class="table table-bordered text-justify">
+              <h4 class="mb-3 font-weight-bold">Total Team Members Required:</h4>
+                <thead>
+                  <tr>
+                    <th>Designation</th>
+                    <th>Needed</th>
+                    <th>Deployed</th>
+                    <th>To Be Deployed</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  @foreach ($designationKeys as $designationName)
+                    <tr>
+                      <td><input type="text" name="designation[]" class="form-control" value="{{ $designations[$designationName]}}" readonly></td>
+                      <td><input type="text" name="needed[{{$designationName}}]" class="form-control" value="{{ ($requirement = $project->getRequirementByDesignation($designationName)) ? $requirement->total_requirement : '' }}"></td>
```

Comment:
This condition looks incorrect. 

1
```suggestion
                      <td><input type="text" name="needed[{{$designationName}}]" class="form-control" value="{{ optional($project->getResourceRequirementByDesignation($designationName))->total_requirement ?? 0 }}"></td>
```